static/elements/chromedash-myfeatures-page.js

```diff
@@ -1,36 +1,53 @@
 import {LitElement, css, html, nothing} from 'lit';
-import './chromedash-approvals-dialog';
```

Comment:
Why remove these two imports?  Because they are imported into components.js and thus available everywhere?  I've gone back and forth on whether each component should follow the principle of "include what you use" or whether components.js should just include all the components that anything uses so that we have one comprehensive list.  What do you think?

Reply:
I added them back in the new commit. I think it makes sense to import the elements to where it's used. 

My thought is that once we have the SPA set up, we can then import only the `*-page.js` components in the component.js, while the other components will only be imported to whichever components that use it!

Reply:
OK, that certainly works.  

It might make sense to have the shoelace-style components imported in components.js because they are widely used and there is a little initialization needed.